Storage/SQL/BrowserDB.swift

```diff
@@ -61,7 +67,15 @@ public class BrowserDB {
         }
 
         // Create or update will also delete and create the database if our key was incorrect.
-        self.createOrUpdate(self.schemaTable)
+        switch self.createOrUpdate(self.schemaTable) {
+        case .Failure:
+            log.error("Failed to create/update the scheme table. Aborting.")
+            fatalError()
```

Comment:
This also ties in a little to [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1317093#c8).